addons/binding/org.openhab.binding.enigma2/src/main/java/org/openhab/binding/enigma2/internal/Enigma2CommandExecutor.java

```diff
@@ -0,0 +1,434 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.enigma2.internal;
+
+import java.io.IOException;
+
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.IncreaseDecreaseType;
+import org.eclipse.smarthome.core.library.types.NextPreviousType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.State;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xml.sax.SAXException;
+
+/**
+ * The {@link Enigma2CommandExecutor} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Thomas Traunbauer - Initial contribution
+ */
+public class Enigma2CommandExecutor {
+    // Source found on
+    // https://dream.reichholf.net/wiki/Enigma2:WebInterface#RemoteControl

```

Comment:
@kaikreuzer yes your right, i just read the wiki page, but did not copy any code from there